configure

```diff
@@ -2,26 +2,18 @@
 # -*- coding: utf-8 -*-
 """configure
 
-Usage: configure <config_file> (--production | --development)
```

Comment:
Some problems when the installation fails. We had to remove everything and start again from scratch. Development mode is better.